package day11.exception.trycatch;

public class RunTimeExample {
	public static void main(String[] args) {
		// 실행예외
		// ArrayIndexOutOfBounds exception
		// int[] arr = {1,2,3};
		// System.out.println(arr[3]);
		
		
		// NumberFormatException
		// String s ="a";
		// int r = Integer.parseInt(s);
		
		// NullPointerException
		// String str =null;
		// str.charAt(0);
		
	}
}
